deploy/konnector/90-deployment.yaml

```diff
@@ -32,4 +32,4 @@ spec:
         readinessProbe:
           httpGet:
             path: "/healthz"
-            port: 8080
+            port: 8090
```

Comment:
This changed due to local `quickstart` not working as backend and konnector uses same port. And its just for probles, so its ok.